python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -925,22 +953,50 @@ def _fast_slow_function_call(
                 except Exception as e:
                     warnings.warn(
                         "The result from pandas could not be computed. "
-                        f"The exception was {e}."
+                        f"The exception was {e}.",
+                        CudfPandasDebugWarning,
                     )
                 else:
                     try:
-                        _assert_fast_slow_eq(result, slow_result)
+                        _assert_fast_slow_eq(result, slow_result, **kwargs)
```

Comment:
Did I remember correctly from another PR that `**kwargs` wasn't needed, or is it needed back in this PR?

Reply:
Yep we decided to remove them